dev/com.ibm.ws.cdi.third.party_fat/fat/src/com/ibm/ws/cdi/thirdparty/tests/ThirdPartyTests.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.thirdparty.tests;
+
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assert.assertThat;
+
+import java.io.File;
+import java.util.List;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+import com.ibm.ws.cdi.thirdparty.apps.deltaspikeWar.DeltaspikeTestServlet;
+import com.ibm.ws.cdi.thirdparty.apps.deltaspikeWar.RequestScopedNumberProvider;
+import com.ibm.ws.cdi.thirdparty.apps.entityListenersEarLibJar.EntityBListener;
+import com.ibm.ws.cdi.thirdparty.apps.entityListenersWar.model.EntityA;
+import com.ibm.ws.cdi.thirdparty.apps.entityListenersWar.web.EntityListenersTestServlet;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import componenttest.topology.utils.HttpUtils;
+
+@RunWith(FATRunner.class)
+public class ThirdPartyTests extends FATServletClient {
```

Comment:
Previously this test was marked `@Mode(FULL)`. However, as it wasn't annotated with `@RunWith(FATRunner)`, the `@Mode` annotation was ignored.